src/cli/impl/StepMachine.cpp

```diff
@@ -60,6 +60,6 @@ void StepMachine::SetContextFromCommandResponse(const CommandResponse& response)
     } else
     {
         if (response.tasks.has_value())
-            context_.GetStorage()->LoadTasks(*response.tasks);
+            *context_.GetStorage() = *response.tasks;
```

Comment:
avoid dereferencing style, especially when you're not checking for nullptr

Reply:
Fixed